Purchases/Misc/RCSystemInfo.m

```diff
@@ -61,10 +61,24 @@ + (NSString *)appVersion {
     return version ?: @"";
 }
 
++ (NSString *)buildVersion {
+    NSString *version = NSBundle.mainBundle.infoDictionary[@"CFBundleVersion"];
+    return version ?: @"";
+}
+
 + (NSString *)platformHeader {
     return self.forceUniversalAppStore ? @"iOS" : PLATFORM_HEADER;
 }
 
+- (nullable NSString *)identifierForVendor {
+#if UI_DEVICE_AVAILABLE
+    return UIDevice.currentDevice.identifierForVendor.UUIDString;
+#elif WKINTERFACE_DEVICE_AVAILABLE
+    return WKInterfaceDevice.currentDevice.identifierForVendor.UUIDString;
+#endif
+    return nil;
```

Comment:
this should be compatible with watchOS, tvOS, and Catalyst (so everything but native mac)

Reply:
I'm not 100% sure about Catalyst, though, since I'm not sure whether it uses the idfv or guid for signing. I'll add support for all Mac on a separate PR, though.

Reply:
this code somewhat duplicates what we have on RCAttributionFetcher, but the cleanup will also come on a different PR for the sake of shipping